schema.sql

```diff
@@ -0,0 +1,17 @@
+drop database if exists bandland;
```

Comment:
I was looking to do this approach initially! Will definitely try this out. This is like how I made my mysql database with a schema.sql file.